Sources/LiveViewNative/Stylesheets/Modifiers/Overrides/SearchScopesModifier.swift

```diff
@@ -22,7 +22,7 @@ struct _SearchScopesModifier<R: RootRegistry>: ViewModifier {
     @ObservedElement private var element
     @LiveContext<R> private var context
     
-    #if os(iOS) || os(macOS) || os(tvOS) || os(xrOS)
+    #if os(iOS) || os(macOS) || os(tvOS) || os(visionOS)
```

Comment:
This is a more reasonable OS name.

Reply:
That was true of earlier betas, but Xcode now warns about `xrOS` not being a known platform. `visionOS` is the correct name.